practice-javascript/src/js/services/connect.js

```diff
@@ -0,0 +1,2 @@
+// import { API } from "../constants/url";
```

Comment:
@thanhtrantruong47 Separate a service file is a good approach, but please help put all CRUD APIs calls into this place.